src/components/demos/read/Modal.js

```diff
@@ -93,10 +105,10 @@ export default function Modal({ update, readDemo, ProgressBar }) {
 
   return (
     <Window data-cy={'modal-main'} styles={theme.modal.window}>
-      <Title title={readDemo.modalTitle} arch={theme.name} />
+      <Title title={readDemo.modalTitle} theme={theme} />
       <Row>
         <Page {...theme.modal.page}>
-          <DemoText data-cy={'modal-main-text'} {...readDemo.txt_col}>
+          <DemoText data-cy={'modal-main-text'} {...readDemo.theme.font}>
```

Comment:
Can text be in white for cheri?

Reply:
<img width="173" alt="Screenshot 2022-09-20 at 16 16 46" src="https://user-images.githubusercontent.com/11794402/191297424-a97296aa-a8aa-45b8-87ce-368ea4894067.png">

Reply:
done